vernac/metasyntax.ml

```diff
@@ -1065,16 +1088,30 @@ module SynData = struct
 
     (* Notation data for parsing *)
 
-    level         : int;
-    syntax_data   : (Id.t * (production_level, production_position) constr_entry_key_gen) list * (* typs    *)
-                    symbol list;                                                                 (* symbols *)
+    level         : level;
+    pa_syntax_data : (Id.t * (production_level, production_position) constr_entry_key_gen) list * (* typs    *)
+                     symbol list;                                                                 (* symbols *)
+    pp_syntax_data : (Id.t * (production_level, production_position) constr_entry_key_gen) list * (* typs    *)
```

Comment:
I suggest giving a name to the inner type (the one before `list`) for documentation purpose.